x-pack/plugins/security_solution/public/timelines/containers/local_storage/index.tsx

```diff
@@ -85,13 +85,29 @@ export const addTimelineInStorage = (
   id: TimelineIdLiteral,
   timeline: TimelineModel
 ) => {
+  const timelineToStore = cleanStorageTimeline(timeline);
   const timelines = getAllTimelinesInStorage(storage);
   storage.set(LOCAL_STORAGE_TIMELINE_KEY, {
     ...timelines,
-    [id]: timeline,
+    [id]: timelineToStore,
   });
 };
 
+const cleanStorageTimeline = (timeline: TimelineModel) => {
+  // discard unneeded fields to make sure the object serialization works
+  const {
+    documentType,
```

Comment:
i think we forget `defaultColumn` or maybe you wanted this way.

Reply:
I did try removing the `defaultColumns` (I guess this one is the one you mean), but doing that it started to appear problems on the _Reset Fields_ action. 
There are several hydration points where this problem would need to be fixed, so I kept the field in the storage as it is the esasiest way to prevent the bug, but we can take another look later on.